tests/kernel/pipe/pipe/src/test_pipe.c

```diff
@@ -874,3 +880,81 @@ void test_pipe_put_min_xfer(void)
 	zassert_equal(res, 0, "did not write min_xfer");
 	zassert_true(bytes_written >= 1, "did not write min_xfer");
 }
+
+/**
+ * @brief Defining and initializing pipes at compile time
+ * @details Check if the kernel provided a mechanism for defining and
+ * initializing pipes at compile time(K_PIPE_DEFINE()).
+ * - Using the pipe which defined and initialized at compile time to
+ * transfer data and verify it.
+ * @ingroup kernel_pipe_tests
+ * @see K_PIPE_DEFINE()
+ */
+void test_pipe_define_at_compileTime(void)
```

Comment:
Call this "test_static_pipe" for clarity, or possibly "test_k_pipe_define".  Also interCapsAreNotZephyrStyle.

Reply:
Hi andyross, thanks for quickly response,  yes,  I found that have the related case in this test suite,  and I removed this one.